design-toolkit/components/src/components/date-field/index.tsx

```diff
@@ -252,10 +181,7 @@ export function DateField<T extends DateValue>({
       </DateInput>
       {shouldShowDescription && (
         <AriaText
-          className={cn([
-            'fg-default-dark text-body-xs empty:hidden',
-            isDisabled && 'fg-disabled',
-          ])}
+          className={`fg-default-dark text-body-xs empty:hidden ${isDisabled && 'fg-disabled'}`}
```

Comment:
All classNames should be in `tv` and all state based styles should be `tv` variants or using variant selectors (in this case the latter)